ext/date/php_date.h

```diff
@@ -37,12 +37,8 @@ PHP_FUNCTION(mktime);
 PHP_FUNCTION(gmmktime);
 
 PHP_FUNCTION(checkdate);
-
-#ifdef HAVE_STRFTIME
 PHP_FUNCTION(strftime);
```

Comment:
As these functions are now always available, we may want to remove some checks from our tests, like https://github.com/php/php-src/blob/852485d8ecd784153e41e565a0a87abf99cf4e0d/ext/date/tests/009.phpt#L6

Reply:
Good one... Yes.